components/filecoin/rs/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "filecoin-cxx"
+description = "FFI Interface to Filecoin"
+version = "0.0.1"
+edition = "2018"
+publish = false
+
+[dependencies]
+base64 = "0.13.0"
+blake2b_simd = "1.0"
+bls-signatures = { version = "0.11.3", default-features = false, features = ["pairing"] }
+cxx = { path = "../../../../third_party/rust/cxx/v1/crate"}
+fvm_shared = { version = "0.6.1 " }
+libsecp256k1 = "0.6"
+num_bigint_chainsafe = { package = "forest_bigint", version = "0.1.2"}
+serde = { version = "^1.0.117", features = ["derive"] }
+serde_cbor = "0.11.1"
```

Comment:
This is flagged as an unmaintained package by `cargo-audit`

Reply:
seems it is not used, removed